packages/atomic/src/components/atomic-result/atomic-result.pcss

```diff
@@ -0,0 +1,18 @@
+@import "tailwindcss/base";
```

Comment:
added really basic styles mostly leveraging variables

Reply:
those are the only styles injected in the result shadow DOM.

Using the tailwind base is kind of a "browser reset"